server/datastore/mysql/mysql.go

```diff
@@ -391,14 +391,19 @@ func sanitizeColumn(col string) string {
 // NOTE: This is a copy of appendListOptionsToSQL that uses the goqu package.
 func appendListOptionsToSelect(ds *goqu.SelectDataset, opts fleet.ListOptions) *goqu.SelectDataset {
 	if opts.OrderKey != "" {
```

Comment:
I wish this kind of format was documented somewhere (at least it should be documented in `ListOptions.OrderKey` field) (e.g. to my surprise while working on a fix I realized `scheduled_queries.platform` is also used similarly, with comma separated values to support multiple platforms).

Reply:
Agreed. It should be a slice probably, but that felt like more work for a hotfix before the release.

Reply:
+1, totally understand, just saying we should add the usage as docs in the field: https://github.com/fleetdm/fleet/blob/c8f3934772bf2766f7a6d1d151fcd9a174dd06c9/server/fleet/app.go#L253-L254